distributeaid/aggregated-public-information
# feat(data-ingestion): Upload surveys from needs data to the NeedsAssessment.Survey collection in Strapi

## What changed?
This pull request populates the `year` and `quarter` field types for the `NeedsAssessment.Survey` collection in Strapi. It relates to task 1 in PR #112. This collection still requires the `needs` relation field type to be populated.

Still to do:

- Populate the `needs` relation field type to complete this `NeedsAssessment.Survey` collection.

## How can you test this?
Add the attached [needs-data.json](https://github.com/user-attachments/files/16826155/needs-data.json) file to the `src/scripts/import-needs-assessment-data` folder, setup the `src/scripts/.env` file, then run the dev server in one terminal and `yarn script:import-needs-assessment-data` in another. The logs should show you that 1 survey has been created successfully or already exists.
